### DISCUSSION TITLE
Add a Notes field

### DISCUSSION THREAD

**Discussion by darylknight** [craftcms/cms#10936] [Ideas] [answered]
It'd be great to add a first party Notes / Instructions field to Craft, specifically for leaving instructions inside a Matrix block, without having to install a third party plugin.

[Entry Instructions](https://plugins.craftcms.com/entry-instructions) was abandoned because of the addition of the UI elements you can add to field Layout Designer, which are great - but you can't add them to matrix blocks.

Specific use-case - I'm creating a page builder with a matrix field, and you have to add at least one field to a block type. All of the current field types require user input, which I'm not going to be displaying:

![image](https://user-images.githubusercontent.com/1250017/163794718-15a6262e-ad49-4b3f-a1db-34f4f5d8bec9.png)

**Comment by brandonkelly** [comment, marked as answer]
I’d much rather add support for UI elements to Matrix fields (#8878) than add a new native field type that has the same purpose.

**Reply by darylknight** [reply]
I agree - that would be great too
